warp/tests/geometry/test_mesh.py

```
@@ -236,6 +270,39 @@ def test_mesh_query_ray(test, device):
         )
 
 
+def test_grouped_mesh_query_ray(test, device):
```

Ruff (0.14.5)

Unused function argument: `test`

(ARG001)